src/utils/auth.ts

```diff
@@ -3,6 +3,9 @@ import * as core from '@actions/core'
 
 import {Context} from '@actions/github/lib/context'
 
+import fs from 'fs'
```

Comment:
This is great and works well, but I'm wondering if we could think about using octokit's `getContents` API: https://octokit.github.io/rest.js/v18#repos-get-content

We do have this as an existing paradigm in the pr labeler: https://github.com/jpmcb/prow-github-actions/blob/main/src/cronJobs/prLabeler.ts#L153-L169

When a user providers a `.github/labels.yaml` or `.github/labels.yml` we dynamically get it via the API to use in the PR labeler. This is nice so we don't have to clone the users repo into their actions runner to get this to work.

Reply:
Hah, sorry I talked myself out of what I just suggested. 😀  I'll switch to retrieving the OWNERS file directly through the API, and if we decide to do multi-owner file support per repo later, we can switch it up.

Not having to clone the repo first is a heck of a lot easier.